ssc/Pos/Ssc/Toss/Logic.hs

```diff
@@ -41,6 +42,8 @@ verifyAndApplySscPayload
         MonadError SscVerifyError m, MonadRandom m)
     => Either EpochIndex (Some IsMainHeader) -> SscPayload -> m ()
 verifyAndApplySscPayload eoh payload = do
+    -- Check the payload for internal consistency.
+    either (throwError . SscInvalidPayload) pure =<< runExceptT (checkSscPayload payload)
```

Comment:
Same as in txp. No reason to use `runExceptT` here.

Reply:
Oh is `Either Text` and instance of `MonadError Text m`? I didn't check.

Reply:
FWIW, we can count on GHC to fuse away the `runExceptT`.

Reply:
Anyway, since `MonadError Text m` will be changed to `Either Text` in functions like `checkSscPayload`, it's good that you removed `runExceptT` here.